### DISCUSSION TITLE
修改LAN IP 后,无法登录web UI

### DISCUSSION THREAD

**Discussion by jaychoww** [immortalwrt/immortalwrt#984] [Q&A]
### Describe the bug

把lan ip改为非192.168.1.1之后,web 自动跳转到登录页面,输入账号密码无法登陆,报 503

### ImmortalWrt version

23.05.0-rc1 (r26503-a5699b283d)

### ImmortalWrt target/subtarget

mediatek/mt7622

### Device

Xiaomi Redmi Router AX6S / Xiaomi Router AX3200

### Image kind

Official downloaded image

### Steps to reproduce

网络-接口-LAN-修改IP
改为192.168.10.1
保存

### Actual behaviour

web跳转到登录页,打开debug模式,所有接口报错503无权限

### Expected behaviour

应该能正常使用

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
1. 检查修改是否被回滚，建议使用强制应用
2. 重启路由器
3. 以隐身模式访问新 ip 地址

**Reply by jaychoww** [reply]
换了个路由又没复现,我先关了这个issue

**Reply by jaychoww** [reply]
补充一下:
从18.05升级(不保留配置)系统后,更改wifi SSID 和 LAN IP,就会出现能登录,其他页面正常,但是访问 http://192.168.106.1/cgi-bin/luci/admin/network/wireless
 页面 和http://192.168.106.1/cgi-bin/luci/admin/network/network
 页面会报 403 错误, 并自动跳到登录页
 清空cookies之后就正常
用匿名模式访问没有这个问题
